cheroot/server.py

```diff
@@ -1130,7 +1130,8 @@ class HTTPServer(object):
     timeout = 10
     """The timeout in seconds for accepted connections (default 10)."""
 
-    version = 'CherryPy/' + cp_version
+    # version = 'CherryPy/' + cp_version
+    version = 'Cheroot/' + __version__
```

Comment:
I'm not sure about branding (whether it should begin with capital `C`, @jaraco should it?), but this looks good :)